.baseline/checkstyle/checkstyle.xml

```diff
@@ -383,6 +383,10 @@
             <property name="format" value="(void setUp\(\))|(void setup\(\))|(void setupStatic\(\))|(void setUpStatic\(\))|(void beforeTest\(\))|(void teardown\(\))|(void tearDown\(\))|(void beforeStatic\(\))|(void afterStatic\(\))"/>
             <property name="message" value="Test setup/teardown methods are called before(), beforeClass(), after(), afterClass(), but not setUp, teardown, etc."/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="!!"/>
```

Comment:
I actually just grepped my home directory for java files containing `!!` and got a ton of matches, all of which I think are harmless (because they're inside strings or comments), e.g.:

```
assertNotNull("TEST_PROJECT should not be null!!!", TEST_PROJECT);
```
```
/* !!!!!!!!!! WARNING !!!!!!!!!!
```

```
return message; // Safe to log!!
```
```
if (!reordered.startsWith("!!")) {
```
```
pqlAssertEquals("2", "(to-lazy-list [1,2,3]) !! 1");
```